site-src/implementations.md

```diff
@@ -123,8 +123,8 @@ See [here][emissary-gateway-api] for more details on using the Gateway API with
 ### Envoy Gateway
 
 [Envoy Gateway][eg-home] is an subproject of the Envoy project for managing an Envoy-based Kubernetes-based application
```

Comment:
@arkodg can you also fix the improper grammar in this PR?

s/an subproject/a subproject/

or reword:

"[Envoy Gateway][eg-home] is a project within the Envoy community for managing Envoy proxy as an application gateway."